packages/react/src/components/LocaleModal/LocaleModal.js

```diff
@@ -162,6 +168,17 @@ const LocaleModal = ({ isOpen, setIsOpen }) => {
   function close() {
     setIsOpen(false);
   }
+
+  /**
+   * @private
```

Comment:
Can you add a description for this private method too?

Reply:
For sure!